x/layer2/types/genesis.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import (
+	crosschaintypes "github.com/functionx/fx-core/v6/x/crosschain/types"
+)
+
+func DefaultGenesisState() *crosschaintypes.GenesisState {
+	params := crosschaintypes.DefaultParams()
+	params.GravityId = "fx-layer2-bridge"
+	params.AverageExternalBlockTime = 2_000
```

Comment:
2s is probably too short

Reply:
We are not sure about layer2's block time.

Reply:
How about setting it high to avoid a timeout, such as 30 minutes @zakir-code